### PR TITLE
@dzucconi => [Show] Add contextual link

### PR DESCRIPTION
**Pull request by mzikherman**
Will add a spec. 

Closes https://artsyproduct.atlassian.net/secure/RapidBoard.jspa?rapidView=4&projectKey=FX&modal=detail&selectedIssue=FX-2190

Screen Shot 2020-10-07 at 4 35 32 PM

Screen Shot 2020-10-07 at 4 35 53 PM

### CODE REVIEW

File: src/v2/Apps/Show/ShowApp.tsx [hunk 1]
```diff
@@ -16,10 +17,26 @@ interface ShowAppProps {
...
+      the <AppContainer /> HOC. */}
+      <Separator as="hr" my={2} />
+    </Box>
+  )
+
   return (
     <>
       <ShowMeta show={show} />
 
+      <ContextualLink />
```
Comment (dzucconi) [inline src/v2/Apps/Show/ShowApp.tsx:38]: non-blocking: I'd just inline this

File: src/v2/Apps/Show/ShowApp.tsx [hunk 2]
```diff
@@ -16,10 +17,26 @@ interface ShowAppProps {
...
+    <Box my={2}>
+      <AppContainer>
+        <HorizontalPadding>
+          <ShowContextualLink show={show} />
+        </HorizontalPadding>
+      </AppContainer>
+
+      {/* This is a full screen width separator, so needs to exist outside of
+      the <AppContainer /> HOC. */}
+      <Separator as="hr" my={2} />
```
Comment (dzucconi) [inline src/v2/Apps/Show/ShowApp.tsx:30]: Makes me think we should just add some support for breaking out of the container in the `Separator` component.

```tsx
<Separator fullScreen />
```
 => 
```css
left: 50%;
margin-left: -50vw;
margin-right: -50vw;
max-width: 100vw;
position: relative;
right: 50%;
width: 100vw;
```

or something

Comment (mzikherman) [reply src/v2/Apps/Show/ShowApp.tsx:30]: I think we should wait to update Palette until this pattern is needed elsewhere.

I like the CSS solution though! Will give that a try and replace it. I couldn't figure out how to get that behavior without breaking out of the `<AppContainer />`.

**Comment by mzikherman** [comment]
Updated with spec.

**Pull request merged**